man/systemctl.xml

```diff
@@ -1502,6 +1502,18 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-P</option></term>
+        <term><option>--property-value=</option></term>
```

Comment:
why the long form? i mean we already have a long form which is "--value --property=". I'd just do the short form as a shortcut for the combination of both long forms. We have done that in the past already at various places, for example systemd-nspawn -U...